### ISSUE TITLE
Ability to provide environment variables via URL parameters

### ISSUE THREAD

**Issue by mikenikles** [GoogleCloudPlatform/cloud-run-button#222]
Hi 👋,

It'd be great if env variables could be provided via URL parameters so that there's no need to commit an `app.json` file to the repository.

As an example, https://github.com/gitpod-io/openvscode-releases contains a `Dockerfile` that needs a `RELEASE_TAG` argument. I'd love to create a Cloud Run button like this:

```markdown
[![Run on Google Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run?git_repo=https://github.com/gitpod-io/openvscode-releases&env=RELEASE_TAG:openvscode-server-v1.60.0)
```

Cheers, Mike

**Comment by jamesward** [comment]
Yes, this would be awesome. Note that right now the only available mechanism to get params from the URL is via the `cloudshell_context` query param which we pares via the `--context` flag: https://github.com/GoogleCloudPlatform/cloud-run-button/blob/c6f7e912d25884bec1b75ca79b91724be5ae13b6/cmd/cloudshell_open/main.go#L76

But the redirector can assemble something to pop into the `cloudshell_context` query param.

**Comment by ahmetb** [comment]
We also have a way of specifying branches/tags/commits. If tags had dockerfiles with pre-populated RELEASE_TAG argument, it would also work.
